pkg/splunk/enterprise/searchheadcluster_test.go

```diff
@@ -1873,3 +1891,99 @@ func TestSearchHeadClusterWithReadyState(t *testing.T) {
 		debug.PrintStack()
```

Comment:
do we need this?

Reply:
yes, its in the test case better to know why it fails with the stack